src/types/PoolKey.sol

```diff
@@ -17,3 +18,25 @@ struct PoolKey {
     /// @notice The hooks of the pool
     IHooks hooks;
 }
+
+library PoolKeyLibrary {
+    /// @notice Returns the unspecified currency and amount for a swap
+    function getUnspecified(PoolKey calldata key, IPoolManager.SwapParams calldata params)
```

Comment:
it's confusing that this function returns the *unspecified* currency but *specified* amount